config/routes.rb

```diff
@@ -0,0 +1,26 @@
+Rails.application.routes.draw do
+  devise_for :users
+  devise_for :users, controllers: { registrations: 'registrations' }
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
- [ ] [OPTIONAL]
Devise routes is duplicated; `devise_for :users`, and this causes an error during project setup. Let's ensure to improve upon the same